app/models/page.rb

```diff
@@ -43,7 +43,8 @@ class Page < ApplicationRecord
   PENDING_TITLE    = 'pending'.freeze
 
   belongs_to :company
-  has_many   :comments, as: :commentable, dependent: :destroy
+  delegate :tenant, to: :company, allow_nil: true
```

Comment:
associations then delegations
keep ordering

Reply:
yes, Roman right. you have already delegate in controller. DON"T make a mess in code. FIX IT in new PR